bin/mail/send-to-mailing-list

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env ruby
```

Comment:
This is just reviving the file we deleted in https://github.com/code-dot-org/code-dot-org/pull/18604